### DISCUSSION TITLE
Home Assistant addon backup

### DISCUSSION THREAD

**Discussion by guitarflipper** [hassio-addons/app-bookstack#224] [Ideas]
I needed a local and easy way to write a library of documentation of all my home network, apps, servers, ...you name it. I found Bookstack as an addon and it's working perfectly.

But, because it's an addon and not running natively in Linux I can't figure out a way to back it up. I've done some searches but no luck so far.

Is there a way to backup and restore?

**Comment by frenck** [comment]
Home Assistant offers backups, found in the setting menu.

**Reply by auggy** [reply]
I was wondering how backups worked for the bookstack add-on and found this post. You should verify the HA backup method works before relying on it. I’ve noticed HA doesn’t backup a lot of the add ons completely and bookstack has a lot of moving parts. SQL, the add-on itself and the media you upload to bookstack.  I’m hoping it’s as easy as using the HA backup system. 🤞🏼

**Reply by jjusepkk** [reply]
I have been using bookstack for a number of years and installed this Home Assistant add-on. It works great.
I was able to restore my DB from the stand alone installation but my search to try and copy my media (images and PDFs) into the Home Assistant add-on has led me here. 
I guess from your comment (and no other replies) that you have checked the HA backup and the media etc are probably not being backed up ?
